samza-core/src/test/java/org/apache/samza/diagnostics/TestDiagnosticsManager.java

```diff
@@ -59,6 +59,14 @@
   private int numPersistentStores = 2;
   private int containerNumCores = 2;
   private boolean autosizingEnabled = false;
+  private String deploymentType = "test deployment type";
```

Comment:
consider adding valid values for deployType and API type, to unit test the util functions you just added